pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -408,7 +416,9 @@ class LightningQubit(Device):
     _CPP_BINARY_AVAILABLE = LQ_CPP_BINARY_AVAILABLE
     _new_API = True
     _backend_info = backend_info if LQ_CPP_BINARY_AVAILABLE else None
-    _config = Path(__file__).parent / "lightning_qubit.toml"
+
+    # This `config` is used in Catalyst-Frontend
+    config = Path(__file__).parent / "lightning_qubit.toml"
```

Comment:
Not know, but I think we need to consider if this will ever be used by Lightning itself or if this will be of exclusive use of Catalyst. If we decide on the latter we should sign it at the file name.

Reply:
The idea was to use this for Lightning Python/C++ APIs at some point in the future. I'll try to draft something in a followup PR to check if it makes sense to use these toml files in Lightning. If not, we can sign it at the file name. Thanks! It's a good suggestion :)

Reply:
Sure! I'm OK with that.